code/src/Card.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import CoverImage from './Cover-image'
+import AlbumName from './Album-name'
+import ArtistName from './Artist-name'
+import data from './data.json';
+import AlbumCoverOverlay from './Album-cover-overlay'
+
+const albumsArray = data.albums.items;
+
+const Card = () => {
+    return (
+        <>
+            {albumsArray.map((album) => {
+                return (
+                    <div className="card" key={album.id}>
```

Comment:
`key` attribute is considered as required only in root element inside loop. We don't have to add it anywhere else. And remember - keys should be unique (if there are actual id's in the data - use that)

This snippet of code is wrong :

```jsx
<div key={someUniqueValue}>
  <h1 key={someUniqueValue}>Heading</h1>
  <p key={someUniqueValue}>Paragraph</p>
  <span key={someUniqueValue}>Span</span>
</div>
```

This snippet of code is correct :

```jsx
 <div key={someUniqueValue}>
  <h1>Heading</h1>
  <p>Paragraph</p>
  <span>Span</span>
</div>
```